### ISSUE TITLE
Right Click on RichTextBlock Crash app in WinUI 3

### ISSUE THREAD

**Issue by ghost1372** [microsoft/microsoft-ui-xaml#7129]
### Describe the bug

Right Click on RichTextBlock Crash app

`Exception thrown at 0x00007FFC5226474C in AlAnvar.exe: Microsoft C++ exception: winrt::hresult_error at memory location 0x000000128DF7C030.
`

### Steps to reproduce the bug

1.Create a new winui 3 app
2.Create listview with RichTextBlock itemtemplate
3.Right Click on RichTextBlock

```
<ListView ItemsSource="{x:Bind Mies}">
            <ListView.ItemTemplate>
                <DataTemplate x:DataType="local:myModel">
                    <StackPanel>
                        <RichTextBlock>
                            <Paragraph>
                                adasd
                            </Paragraph>
                        </RichTextBlock>
                    </StackPanel>
                </DataTemplate>
            </ListView.ItemTemplate>
        </ListView>
```

### NuGet package version

WinUI 3 - Windows App SDK 1.1 Preview 3: 1.1.0-preview3

### Windows app type

- [X] UWP
- [X] Win32

### Device form factor

Desktop

### Windows version

Windows 11 (21H2): Build 22000

**Comment by kmgallahan** [comment]
Dupe of #4804 (applies to any control that uses a TextBox internally)

**Comment by ghost1372** [comment]
This issue is fixed in WASDK 1.1.0 stable
